### PR TITLE
[UI] Add options to sort games by title and status

### PR DESCRIPTION
**Pull request by flavioislima**
https://user-images.githubusercontent.com/26871415/165805498-c70e0951-9ef6-45e0-bf27-3b26403fa4d4.mp4

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by arielj — COMMENTED** [review]
This is happening to me:
image

The sort order applies to the first installed order, so I see A-Z in installed then Z-A in not-installed. If I toggle the installed state sort, I get A-Z not-installed and then Z-A installed.

I imagine there's a bug in my original sort by state code (the one that only seem to apply in the build (?)).


A question on this is if we should store the sorting in the config, now the change is not persisted so if you change to another screen or restart heroic it falls back to the default A-Z installed first

File: src/screens/Library/index.tsx [hunk 1]
```diff
@@ -68,7 +70,12 @@ export default function Library(): JSX.Element {
     return (
       <div className="titleWithIcons">
         {getLibraryTitle(category, filter, t)}
-        <ActionIcons />
+        <ActionIcons
+          sortAscending={sortAscending}
+          setSortAscending={() => setSortAscending(!sortAscending)}
```
[the remark this answers is not in the document]
Comment (flavioislima) [reply src/screens/Library/index.tsx:75]: Yes, sounds good.

**Comment by flavioislima** [comment]
Yes, I will experiment a bit more with this logic tomorrow and add the persistence on the store. Makes sense to get back to same filter after restarting heroic and changing screens. 👍

**Comment by flavioislima** [comment]
@arielj can you try again? I believe I could fix the logic now.

**Review by arielj — COMMENTED** [review]
When the sort shows A->Z, the games are sorted Z->A. The order is consistent for installed/not installed now.

image

Maybe this is intentional though, like if the icon means `click here to sort from A->Z` instead of `the current order is from A->Z, click here to toggle it`.

If this was intentional I vote to change it to mean `the current order is ...`, I think it's more clear, and the yellow color gives the idea that that is `the current` state of that action (like the layout icons and the installed sort icon)

**Comment by flavioislima** [comment]
Yes, I thought would he ideal to show what's next after clicking but that might he confusing as well.

**Review by arielj — APPROVED** [review]
Looks good!

**Pull request merged**